canonicalwebteam/blog/common_view_logic.py

```diff
@@ -42,13 +42,14 @@ def get_index(self, page=1, category="", enable_upcoming=True):
                     categories=[events["id"], webinars["id"]],
                 )
 
-        articles, total_pages = api.get_articles(
+        articles, metadata = api.get_articles(
             tags=self.tag_ids,
             tags_exclude=self.excluded_tags,
             exclude=[article["id"] for article in featured_articles],
             page=page,
             categories=[category_id],
         )
+        total_pages = metadata["total_pages"]
```

Comment:
this is done twice (also in the get_index_context if I am nto wrong)

Reply:
on line 57 the `total_pages` gets passed into  `get_index_context`. Shouldnt happen twice :)